rpa-bot/1.1/modules/ROOT/pages/configuration.adoc

```diff
@@ -55,24 +55,22 @@ Format: `+http://<Serveraddress>:<Port>+`, for example: `+http://myproxy.de:881+
 [[secure-session]]
 == Secure Session
 
-Enable the bot to use Secure Sessions by entering credentials the bot uses to login to the computer and to open a hidden session. If you don't add credentials here, the bot can only use a running desktop session.
+Enable the bot to use secure sessions by entering credentials the bot uses to login to the computer and to open a hidden session. If you don't add credentials here, the bot can only use a running desktop session.
 
 image:bot-configurator-secure-session.png[The RPA Bot configurator application showing the Secure Session settings panel, 50%, 50%]
 
 * *Default Secure Session User*
 +
-If you want the bot to use Secure Sessions, enter the name of the user the bot uses to login to the computer and to run a hidden session. If the environment of the computer the bot is installed on is embedded in a domain and you want to use a local user, write the user name as follows: `.\username`. For domain users, use the following form: `mydomain\username`.
+If you want the bot to use Secure Sessions, enter the name of the user the bot uses to log in to the computer and to run a hidden session. If the environment of the computer the bot is installed on is embedded in a domain and you want to use a local user, write the user name as follows: `.\username`. For domain users, use the following form: `mydomain\username`.
 * *Activate certificate based login on SSL enabled terminal servers*
 +
 Enable this option if you are installing the bot on a terminal server that only allows SSL protected RDP connections.
 * *Handle “Always prompt client for password” option*
 +
-There is a Group Policy called `fPromptForPassword`. If this policy is enabled, the password has to be entered a second time, when the RDP session is established. If this is the case, enable this option. Also choose the correct form of dialog. On some machines the second password entry is requested by a separate window in form of
-a popup dialog. And on other machines it is an embedded screen like the normal windows login.
-//To determine if the option as to be used and which form of dialog needs to be configured, it is a good idea to execute the steps described in chapter Functional test.
+There is a Group Policy called `fPromptForPassword`. If this policy is enabled, you must enter the password a second time, when the RDP session is established. If this is the case, enable this option. Also, choose the correct form of dialog. Some machines request the second password entry via a popup dialog. Other machines embedd the prompt into the same screen, like the regular Windows login.
```

Comment:
if what is the case?

Reply:
embedd -> embed

Reply:
not sure about using "regular" in "regular Windows login," but people probably know what we mean.